lib/Act/Manual/Developer/UrlsAndFiles.pod

```diff
@@ -0,0 +1,191 @@
+for the URL with C</index.html> appended.
+
+=item The Conference HTML pages
+
+URLs of Conference HTML pages start with the URL of the conference
+homepage and end in C<.html>.  Beyond C<index.html>, each conference
+can have its own set of URLs.
+
+=item The Conference Statics
+
+Each conference may have its own set of images, style sheets, and
+JavaScript sources - and probably more.  They are served from
+subdirectories directly below the conference homepage.  So, they share
+the paths with the conference HTML pages.  Act uses the convention
+that paths ending in C<.html> are conference HTML pages, all the
+others are static pages which are just passed through from
+corresponding files.  These files are provided in the directory
+C<wwwdocs> of the conference repositories on GitHub.
+
+=item Act Services for Conferences
+
+A third set of URLs directly below the conference homepages is
+occupied by Act services like e.g. C<register> to register a new user,
+C<talks> to see the talks for this conference, and many more.  These
+services also collide with the namespace of conference statics, and
+the author has not yet tried which one takes precedence if there's
+e.g. a directory C<users> under C</wwwdocs>.
+
+=back
+
+=head2 Files provided by Act
+
+In addition to the Act software, the distribution contains a set of
+directories from which pages are either served directly, or constructed.
+
+=over
+
+=item C<conferences>
+
+This directory came up with the dockerized Act implementation and may
+not be present in your installation.
+
+=item C<templates>
+
+This directory holds the templates for
+L<Template Toolkit|http://tt2.org> which is used by Act to convert the
+texts provided by the conferences into full-blown web pages.
+
+See L<Act::Manual::Developer::Templates> for more information.
+
+=item C<wwwdocs>
+
+These directories under C<wwwdocs> are served directly under the root
+URL, without further processing.  They contain some images, a pretty
+outdated version of jquery, and some stylesheets.
+
+Conference organisers may use these documents, or commit their own,
+using the appropriate links.
+
+=back
+
+=head2 Files provided by Act conferences
+
+=head3 How Organizers deliver their files
+
+See L<Act::Manual::Organizer::Layout>! Act conferences run on the
+original site are maintained by organizers on GitHub under
+L<https://github.com/Act-Conferences>.  There's a lot of history
+there, that's why the directory structure as described in this page
+can't be changed without disruption.
+
+=head3 Where Act is looking for them
+
+I<Disclaimer:> The author doesn't know the directory structure on any
+productive Act server.  The following description is based on the PSGI
+based implementation, so may be inaccurate.
+
+Act has a "home" directory which is passed as a mandatory environment
+variable C<ACTHOME> to the engine.  All paths are calculated from there.
+We are writing C<"$home"> here for this directory.  Every conference
```

Comment:
🛑 I do not see any reason to change that naming scheme here, $ACTHOME is a perfectly understood concept within those ever worked on this code

Reply:
Fair enough - done and committed.